src/main/resources/stashpullrequestbuilder/stashpullrequestbuilder/StashBuildTrigger/help-needsWorkOnBuildFailure.html

```diff
@@ -0,0 +1,4 @@
+<div>
+  Tell Stash to automatically marked the PR as needs work if the build job has been
```

Comment:
marked -> mark
"Needs Work" should be quoted or emphasized somehow and possibly capitalized.

Reply:
done